src/uct/ib/ud/base/ud_ep.c

```diff
@@ -684,6 +684,35 @@ static uct_ud_ep_t *uct_ud_ep_create_passive(uct_ud_iface_t *iface, uct_ud_ctl_h
     return ep;
 }
 
+static void uct_ud_ep_rx_ctl_drop_packet(uct_ud_ep_t *ep, uct_ud_neth_t *neth)
+{
+    uct_ud_ctl_hdr_t *ctl = (uct_ud_ctl_hdr_t*)(neth + 1);
+    int creq_received     = (ep->flags & UCT_UD_EP_FLAG_CREQ_RCVD) &&
+                            (ep->rx_creq_count > 1);
+    int crep_received     = ep->flags & UCT_UD_EP_FLAG_CREP_RCVD;
+
+    ucs_assert((ctl->type == UCT_UD_PACKET_CREQ) ||
+               (ctl->type == UCT_UD_PACKET_CREP));
+
+    ucs_trace_data("ep %p: drop %s with psn %u, head_sn %u",
+                   ep, (ctl->type == UCT_UD_PACKET_CREQ) ? "CREQ" : "CREP",
```

Comment:
we can pass "CREP" or "CREQ" string as a parameter (to simplify code)

Reply:
done